### PR TITLE
Vite migration

### PR DESCRIPTION
**Pull request by kurund**
- [x] Migrate to Vite
- [x] Fix dev mode
- [x] Fix cypress test building
- [x] Unit test fixes
- [x] CI fixes
- [x] Build process fixes
- [ ] eslint fixes
- [x] Deepscan fixes
- [x] Errors when we put logflare key in env
- [x] CSS compose issue

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,34 +1,28 @@
...
+      "@testing-library/jest-dom",
+      "vite-plugin-svgr/client"
     ],
-    "typeRoots": [
-      "./src/types",
-      "./node_modules/@types"
-    ],
-    "allowJs": true,
+    "typeRoots": ["./src/types"],
+    "target": "es2022",
```
Comment (mdshamoon) [inline tsconfig.json:10]: Should we have target es5 as the support for es2022 will be less I think

**Review by mdshamoon — COMMENTED** [review]
Not sure why but lesser integration tests are running in this branch. Checking the cypress branch

File: tsconfig.json (hunk 1 above)
Comment (kurund) [reply tsconfig.json:10]: We should target `ESNext `

Comment (kurund) [reply tsconfig.json]: I have updated the config.

**Comment by mdshamoon** [comment]
@kurund added all the environment variables and deployed on staging. Now getting this error when I open chat screen.
image
I have added stream polyfill but somehow still not working. Checking other solutions for this.

**Review by mdshamoon — APPROVED** [review]
LGTM 👍.

**Pull request merged**